StackExchange.Redis/StackExchange/Redis/ExponentialRetry.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+
+namespace StackExchange.Redis
+{
+
+    public class ExponentialRetry : IReconnectRetryPolicy
```

Comment:
Needs XML comments, telling people how this works, since it'll appear directly in their code.